README.md

```
@@ -1,2 +1,146 @@
+    |  |   Initramfs Builder | |
+    |  | Kernel + Runtime    | |
+    |  | Dependencies        | |
+    |  +--------------------+  |
+    +--------------------------+
+
+### Initramfs Builder
+
+The initramfs builder handles all the process of creating the image that will be used to start the VM and customize it.
+It will bring dependancies for languages
+
+**Features :**
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)